### PR TITLE
Updating language for ES pricing

### PR DESCRIPTION
**Pull request by soutenniza**
## Changes proposed in this pull request:
- Update language for ES

:sunglasses:[PREVIEW URL](https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/updating-pricing)

## Security Considerations
None just language updates

### CODE REVIEW

**Comment by mheadd** [comment]
@melanienleopold I finally got the [new section to look right](https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/updating-pricing/pricing/). Let me know what you think.

File: _pages/pricing.md [hunk 1]
```diff
@@ -150,6 +150,15 @@ redirect_from:
           </li>
         </ul>
       </div>
+    </div>
+  </section>
+    <section class="usa-section">
```
Comment (apburnes) [inline _pages/pricing.md]:
```suggestion
  <section class="usa-section">
```

**Review by apburnes — COMMENTED** [review]
A couple spaces.

**Review by mheadd — APPROVED** [review]
Looks good!

**Pull request merged**